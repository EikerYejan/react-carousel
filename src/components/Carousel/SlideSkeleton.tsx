import styled, { keyframes } from 'styled-components'
import { themeColor } from '../../theme'

const opacitySkeleton = keyframes`
    0% {
      opacity: 1;
    }
    50% {
      opacity: 0.7;
    }
    100% {
      opacity: 1;
    }
`

export default styled.div`
  width: 100%;
  height: 100%;
  background: ${themeColor('dark')};
  animation: ${opacitySkeleton} 1s linear infinite forwards;
  position: relative;
`
