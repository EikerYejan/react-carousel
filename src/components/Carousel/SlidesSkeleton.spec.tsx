import { render } from '@testing-library/react'
import SlideSkeleton from './SlideSkeleton'

describe('SlideSkeleton', () => {
  it('Should render', () => {
    const { baseElement } = render(<SlideSkeleton />)

    expect(baseElement).toMatchSnapshot()
    expect(baseElement).toBeInTheDocument()
  })
})
