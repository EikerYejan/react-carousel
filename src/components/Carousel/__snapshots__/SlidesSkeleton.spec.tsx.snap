// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SlideSkeleton Should render 1`] = `
.c0 {
  width: 100%;
  height: 100%;
  -webkit-animation: cKAanV 1s linear infinite forwards;
  animation: cKAanV 1s linear infinite forwards;
  position: relative;
}

<body>
  <div>
    <div
      class="c0"
    />
  </div>
</body>
`;
